test/lib/y2system_role_handlers/dashboard_role_finish_test.rb

```diff
@@ -43,6 +43,7 @@
         handler.run
         records = ntp_conf.records.select { |r| r.type == "server" }
         expect(records.map(&:value)).to eq([ntp_server])
+        expect(records.first.options).to eq(["iburst"])
```

Comment:
Um, I find it a bit inconsistent, on the previous lines you check *all* record values, but here you check only the first item.

Why not using something similar here? E.g. `expect(records.map(&:options)).to eq([["iburst"]])`